### PR TITLE
[ATS 2018] Semifinals update

### PR DESCRIPTION
**Pull request by fajar13k**
### Description
- Updates tournament to Semifinals stage (Pools, Schedules, Results) along with its translation

### Status
WIP

### CODE REVIEW

File: wiki/Tournaments/ATS/2018/en.md [hunk 1]
```diff
@@ -40,19 +40,19 @@ The **Asian Taiko Showdown 2018 _(ATS 2018)_**, is a double elimination one-on-o
...
-| Statisticians | ![][flag_ID] [fajar13k](https://osu.ppy.sh/users/7100002), ![][flag_ID] [Aozora-](https://osu.ppy.sh/users/6918206), ![][flag_US] [JDrago14](https://osu.ppy.sh/users/7690078) |
+| --- | --- |
+| Organizers | ![][flag_ID] [fajar13k](/users/7100002), ![][flag_US] [JDrago14](/users/7690078) |
+| Map Selectors | ![][flag_ID] [Lightning Wyvern](/users/1533122), ![][flag_ID] [Guardistack-](/users/1602428), ![][flag_MY] [cdhsausageboy](/users/2403621) |
+| Commentators | ![][flag_AU] [Jaye](/users/4841352), ![][flag_ID] [Niko-nyan](/users/906991), ![][flag_CA] [janitoreihil](/users/3307897), ![][flag_US] [Das](/users/3165416) |
+| Streamers |  ![][flag_JP] [SakuraFrost-](/users/8147993), ![][flag_ID] [rap12](/users/2300279) |
```
Comment (Joehuu) [inline wiki/Tournaments/ATS/2018/en.md]: Remove the added space.

File: wiki/Tournaments/ATS/2018/en.md [hunk 2]
```diff
@@ -121,127 +121,171 @@ The Asian Taiko Showdown 2018 is run by various osu!taiko community members.
...
+### Semifinals
+
+**[Download the mappack here! (81MB)](https://drive.google.com/file/d/17XmIGdwsdxLNYnkc7EoL0mLczF8VoUsw/view?usp=sharing)**
+
+- NoMod
+  - [Mitsuyoshi Takenobu no Ani - Amphisbaena (HiroK) \[Inner Oni\]](/beatmapsets/737973/#taiko/1557391)
+  - [Sampling Masters MEGA - Kattobi KEIKYU Rider (7_7) \[Gatagoto Gatagoto Oni\]](/beatmapsets/673769/#taiko/1425840) 
+  - [MASAKI - Scarlet Lance (m1kado) \[Inner Oni\]](/beatmapsets/681440/#taiko/1440822) 
+  - [Camellia feat. nanahira - Kizuitara Shunkashuutou (shinchikuhome) \[Taiko Inner Oni\]](/beatmapsets/532994/#taiko/1129200)
+  - [Camellia - /*decryption(instrumental)*/ (Nifty) \[Inner Oni\]](/beatmapsets/674049/#taiko/1426341)
```
Comment (WalterToro) [inline wiki/Tournaments/ATS/2018/en.md]: You might need to escape the italics here

File: wiki/Tournaments/ATS/2018/en.md [hunk 3]
```diff
@@ -121,127 +121,171 @@ The Asian Taiko Showdown 2018 is run by various osu!taiko community members.
...
+  - [YUC'e - Chemical Cookie (KinomiCandy) \[ANOTHER ONI\]](/beatmapsets/747516/#taiko/1575117)
 - Hidden
-  - [Mystic Moon - The Zoo Zone (cdhsausageboy) \[Oni (ATS Ver.)\]](https://osu.ppy.sh/beatmapsets/795265/#taiko/1670210) 
-  - [Ayasa - Kou ! Tetsu ! Ketsu ! Sen ! -Ironbottom Sound- (S a n d) \[Inner Oni\]](https://osu.ppy.sh/beatmapsets/519566/#taiko/1103735) 
+  - [Mystic Moon - The Zoo Zone (cdhsausageboy) \[Oni (ATS Ver.)\]](/beatmapsets/795265/#taiko/1670210) 
+  - [Ayasa - Kou ! Tetsu ! Ketsu ! Sen ! -Ironbottom Sound- (S a n d) \[Inner Oni\]](/beatmapsets/519566/#taiko/1103735) 
 - HardRock
-  - [Project Grimoire - Caliburne ~Story of the Legendary sword~ (-xNaCLx-) \[Saltaiko@NaCL\]](https://osu.ppy.sh/beatmapsets/326711/#taiko/725333) 
-  - [ginkiha + kamome sano - Dawn of Asia (TKS) \[Inner Oni\]](https://osu.ppy.sh/beatmapsets/446239/#taiko/958486) 
+  - [Project Grimoire - Caliburne ~Story of the Legendary sword~ (-xNaCLx-) \[Saltaiko@NaCL\]](/beatmapsets/326711/#taiko/725333)
```
Comment (WalterToro) [inline wiki/Tournaments/ATS/2018/en.md]: Another escaping here

File: wiki/Tournaments/ATS/2018/en.md [hunk 4]
```diff
@@ -121,127 +121,171 @@ The Asian Taiko Showdown 2018 is run by various osu!taiko community members.
...
 
 **[Download the mappack here! (105 MB)](https://drive.google.com/file/d/1dJbbGVljsn9AYSHsubbzfcmBc0cwPPh0/view?usp=sharing)**
 
 - NoMod
-  - [Kamui Gakupo, KAITO, Kagamine Len - Immoral Memory~The Lost Memory~ (7odoa) \[Taiko Oni\]](https://osu.ppy.sh/beatmapsets/44898/#taiko/145660)
-  - [dors k(uro) - dreeeeam (Vulkin) \[Inner Oni\]](https://osu.ppy.sh/beatmapsets/788077/#taiko/1661560) 
-  - [Haywyre - Insight (Guardistack-) \[Mini Satanic\]](https://osu.ppy.sh/beatmapsets/739890/#taiko/1561061) 
-  - [Inspector K - Disconnected Hardkore (CanBlaster Remix) (SE_Soul) \[SE_Soul Crazy Taiko\]](https://osu.ppy.sh/beatmapsets/88094/#taiko/240302)
-  - [The Ghost Of 3.13 - Forgotten (\[luanny_phnyx\]) \[phNyx's Taiko Oni\]](https://osu.ppy.sh/beatmapsets/55560/#taiko/168807)
+  - [Kamui Gakupo, KAITO, Kagamine Len - Immoral Memory~The Lost Memory~ (7odoa) \[Taiko Oni\]](/beatmapsets/44898/#taiko/145660)
```
Comment (WalterToro) [inline wiki/Tournaments/ATS/2018/en.md]: And another escaping here.

**Pull request merged**